source/acquaman/AMGenericStepScanConfiguration.cpp

```diff
@@ -197,12 +200,16 @@ void AMGenericStepScanConfiguration::setControl(int axisId, AMControlInfo newInf
 		connect(scanAxisAt(1)->regionAt(0), SIGNAL(regionStepChanged(AMNumber)), this, SLOT(computeTotalTime()));
 		connect(scanAxisAt(1)->regionAt(0), SIGNAL(regionEndChanged(AMNumber)), this, SLOT(computeTotalTime()));
 		connect(scanAxisAt(1)->regionAt(0), SIGNAL(regionTimeChanged(AMNumber)), this, SLOT(computeTotalTime()));
+
+		emit axisControlInfoAdded();
+		emit axisControlInfoChanged();
 	}
```

Comment:
I saw potential of code generalization here...
codes to add / replace axis 0 and 1 are very alike